### DISCUSSION TITLE
映射到本地磁盘后显示的不是云盘的容量与占用，而是本地C盘的，这个什么问题？

### DISCUSSION THREAD

**Discussion by han-xiaowang** [messense/aliyundrive-webdav#113] [Q&A]
小米AC2100路由器，编译的Lean's OpenWrt，安装的ipk

**Comment by messense** [comment]
截图？

**Comment by han-xiaowang** [comment]
![微信截图_20211103162413](https://user-images.githubusercontent.com/13508028/140042970-9d31ab37-4eb6-4587-b7e9-1e2802ea7e01.png)

**Reply by messense** [reply]
不清楚为啥，不影响使用的话忽略吧。。。

**Comment by han-xiaowang** [comment]
![微信截图_20211103181809](https://user-images.githubusercontent.com/13508028/140043318-804acf03-534b-4e3f-b428-7c535507ec9b.png)
